packages/decoder/tsconfig.json

```diff
@@ -10,14 +10,8 @@
     "sourceMap": true,
     "outDir": "dist",
     "baseUrl": ".",
-    "lib": ["es2019"],
+    "lib": ["es2019","dom"],
```

Comment:
Related to [previous comment](https://github.com/trufflesuite/truffle/pull/6013/files#r1184528770)